### PR TITLE
add partial delete functionality

### PR DESCRIPTION
**Pull request by kyrikoni**

### CODE REVIEW

File: src/components/Comments.jsx [hunk 1]
```diff
@@ -44,6 +50,35 @@ export const Comments = ({ singleReview }) => {
     }
   };
 
+  const handleDeleteComment = (comment_id) => {
+    getUsers()
+      .then((users) => {
+        console.log(users);
```
Comment (Poonam-raj) [inline src/components/Comments.jsx]: Get rid of this console log!

File: src/components/Comments.jsx [hunk 2]
```diff
@@ -44,6 +50,35 @@ export const Comments = ({ singleReview }) => {
...
+        return users;
+      })
+      .then((users) => {
+        if (users.username !== comments.author) {
+          alert("Cannot delete another users comment");
```
Comment (Poonam-raj) [inline src/components/Comments.jsx]: This alert didn't pop up for me and I could still click delete, you need to completely stop a user from even being able to start trying to do that

**Review by Poonam-raj — CHANGES_REQUESTED** [review]
Spamming delete: One thing I would really like to see here is to stop a user clicking delete multiple times if their internet is slow (like the post functionality)

Other user comments: I could happily delete the comment of another user, please make sure deletion isn't even a option for other comments.

So some changes needed but really nice so far, i like how you remove the deleted comment using a filter and the clean UI for it all

File: src/components/Comments.jsx (hunk 2 above)
Comment (kyrikoni) [reply src/components/Comments.jsx]: How do I even set a user though if we haven't set up any login user authentication?

**Pull request merged**